LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -149,6 +149,28 @@ private String valueOf(final Object obj) {
 
     @Override
     public ILabAccount createTempAccount(@NonNull final TempUserType tempUserType) throws LabApiException {
+        // Adding a second attempt here, api sometimes fails to create the temp user.
```

Comment:
What specific error does it give when it fails to create the temp user?

Reply:
`FAILED_TO_CREATE_TEMP_USER`